docs/TESTS.md

```diff
@@ -32,52 +49,38 @@ Tests are run against an actual backend so they require a valid backend address
 1. `TFC_RUN_TASK_URL` - Run task integration tests require a URL to use when creating run tasks. To learn more about the Run Task API, [read here](https://www.terraform.io/cloud-docs/api-docs/run-tasks)
 
-You can set your environment variables up however you prefer. The following are instructions for setting up environment variables using [envchain](https://github.com/sorah/envchain).
-   1. Make sure you have envchain installed. [Instructions for this can be found in the envchain README](https://github.com/sorah/envchain#installation).
-   1. Pick a namespace for storing your environment variables. I suggest `go-tfe` or something similar.
-   1. For each environment variable you need to set, run the following command:
-      ```sh
-      envchain --set YOUR_NAMESPACE_HERE ENVIRONMENT_VARIABLE_HERE
-      ```
-      **OR**
-    
-      Set all of the environment variables at once with the following command:
-      ```sh
-      envchain --set YOUR_NAMESPACE_HERE TFE_ADDRESS TFE_TOKEN GITHUB_TOKEN GITHUB_POLICY_SET_IDENTIFIER
-      ```
-
-### 3. Make sure run queue settings are correct
+## 3. Make sure run queue settings are correct
 
 In order for the tests relating to queuing and capacity to pass, FRQ (fair run queuing) should be
 enabled with a limit of 2 concurrent runs per organization on the Terraform Cloud or Terraform Enterprise instance you are using for testing.
 
-### 4. Run the tests
+## 4. Run tests
 
-#### Running all the tests
-As running the all of the tests takes about ~20 minutes, make sure to add a timeout to your
-command (as the default timeout is 10m).
+For most situations, it's recommended to run specific tests because it takes about 20 minutes to run all of the tests.
 
-##### With envchain:
+### Running specific tests
+
+Typically, you'll want to run specific tests. The commands below use notification configurations as an example.
+
+#### With envchain:
 ```sh
-$ envchain YOUR_NAMESPACE_HERE go test ./... -timeout=30m -tags=integration
+$ envchain YOUR_NAMESPACE_HERE go test -run TestNotificationConfiguration -v ./... -tags=integration
 ```
 
-##### Without envchain:
+#### Without envchain:
 ```sh
-$ TFE_TOKEN=xyz TFE_ADDRESS=xyz ENABLE_TFE=1 go test ./... -timeout=30m -tags=integration
+$ TFE_TOKEN=xyz TFE_ADDRESS=xyz ENABLE_TFE=1 go test -run TestNotificationConfiguration -v ./... -tags=integration
 ```
 
-#### Running specific tests
-
-The commands below use notification configurations as an example.
+### Running all tests
+It takes about 20 minutes to run all of the tests, so specify a larger timeout when you run the tests (_the default timeout is 10 minutes_):
 
-##### With envchain:
+#### With envchain:
 ```sh
-$ envchain YOUR_NAMESPACE_HERE go test -run TestNotificationConfiguration -v ./... -tags=integration
+$ envchain YOUR_NAMESPACE_HERE go test ./... -timeout=30m -tags=integration
```

Comment:
I assume the timeout was added for consistency.

Reply:
The timeout was in the previous version of the document too - it's needed if one wants to run all of the tests because it takes 20 minutes to run all tests.